doc/ale-solidity.txt

```diff
@@ -0,0 +1,16 @@
+===============================================================================
+ALE Solidity Integration                                       *ale-solidity-options*
```

Comment:
Your tags are poorly aligned. They should be aligned to the right margin. `:ALEFixSuggest` will suggest a fixer for fixing this easily.